blockchain/accept.go

```diff
@@ -90,3 +92,84 @@ func (b *BlockChain) maybeAcceptBlock(block *btcutil.Block, flags BehaviorFlags)
 
 	return isMainChain, nil
 }
+
+// maybeAcceptBlockHeader potentially accepts the header to the block index and,
+// if accepted, returns the block node associated with the header.  It performs
+// several context independent checks as well as those which depend on its
+// position within the chain.  It should be noted that some of the header fields
```

Comment:
Which header fields require the full block data to be able to validate? This looks like a exact copy paste of the comments from dcrd.

Reply:
Yeah I will remove that part.
The header fields that they are talking about are some data about voters and stakes, which we are not maintaining here